### DISCUSSION TITLE
Albanian channels

### DISCUSSION THREAD

**Discussion by blendberila** [iptv-org/community#974] [Q&A]
Can anyone help to get more albanian channels like some sports channels 
Or sombody sent me some files with albanian channels
Thanks.

**Comment by BellezaEmporium** [comment]
The only "clue" I could tell you is : look for deleted streams that were in al.m3u around a year ago. Just change the server name to something like "eu" and watch the magic happen :).

**Comment by freearhey** [comment]
https://iptv-org.github.io/iptv/countries/al.m3u
